tests/orbits/rebound_test.py

```diff
@@ -73,6 +73,80 @@ def test_keplerian(orbit):
         assert np.allclose(y, y0)
         assert np.allclose(z, z0)
 
+def test_gr_orbit_low_mass_limit():
+    from exoplanet.theano_ops.rebound import ReboundOp as test_rebound_op
+    t = np.linspace(0, 40, 200)
+    m_planet = 0.01
+    m_star = 1
+    period = 100.
+    orbit = ReboundOrbit(
+        m_star=m_star,
+        r_star=1.0,
+        t0=0.5,
+        period=period,
+        ecc=0.1,
+        omega=0.5,
+        Omega=1.0,
+        incl=0.25 * np.pi,
+        m_planet=m_planet,
+    )
+
+    orbit_gr = ReboundOrbit(
+        m_star=m_star,
+        r_star=1.0,
+        t0=0.5,
+        period=period,
+        ecc=0.1,
+        omega=0.5,
+        Omega=1.0,
+        incl=0.25 * np.pi,
+        m_planet=m_planet,
+        ReboundOp=test_rebound_op,
```

Comment:
What's the point of having this here? Is this so that `rebound_pymc3` doesn't get used?

Reply:
Yep! I took inspiration from the [`theano_ops` testing](https://github.com/exoplanet-dev/exoplanet/blob/1bc753c4b4df3ad1a2217aa7fc29623fdd059ea7/tests/theano_ops/rebound_op_test.py).